packages/atomic/src/locales.json

```
@@ -1185,15 +1193,21 @@
   },
   "with-colon": {
     "en": "{{text}}:",
-    "fr": "{{text}}\u00A0:"
+    "fr": "{{text}}\u00A0:",
+    "zh-CN":"{{text}}：",
```

there's a missing space before the string here.
```suggestion
    "zh-CN": "{{text}}：",
```